core/example/solvers/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_executable(SPME particles.cpp direct.cpp ewald.cpp pme.cpp main.cpp)
+target_link_libraries(SPME cabanacore fftw3)
```

Comment:
Where is the `fftw3` target defined?

Reply:
@junghans I am surprised this did not break at configuration time on the CI.

Reply:
@dalg24 cmake allows to put in random library string, but it will then only fail at compile/link time.

Reply:
```suggestion
find_package(FFTW3 REQUIRED)
target_link_libraries(SPME cabanacore ${FFTW3_LIBRARIES})
```

Reply:
Does FFTW3 actually provide a CMake configuration file or do we need to add a find module?

Reply:
From what I gather, we will need to add a find module for FFTW3.

Reply:
Just use https://github.com/votca/tools/blob/master/CMakeModules/FindFFTW3.cmake

Reply:
Thanks @junghans !